test/conformance/virtual_memory/urPhysicalMemRelease.cpp

```diff
@@ -8,8 +8,23 @@ using urPhysicalMemReleaseTest = uur::urPhysicalMemTest;
 UUR_INSTANTIATE_DEVICE_TEST_SUITE_P(urPhysicalMemReleaseTest);
 
 TEST_P(urPhysicalMemReleaseTest, Success) {
+    uint32_t referenceCount = 0;
+    ASSERT_SUCCESS(
+        urPhysicalMemGetInfo(physical_mem, UR_PHYSICAL_MEM_INFO_REFERENCE_COUNT,
+                             sizeof(referenceCount), &referenceCount, nullptr));
+    ASSERT_EQ(referenceCount, 1);
```

Comment:
Is the implementation allowed to have internal references to the memory object that will inflate this number? I don't think the spec says anywhere that `UR_PHYSICAL_MEM_INFO_REFERENCE_COUNT` corresponds to the number of externaly-to-ur visible handles nor that it starts at 0. Of course, I could be missing something in the spec.

Reply:
This should be checking `ASSERT_GE(1, referenceCount)` in case the implementation does to additional retain/release.

Reply:
Updated this.